javascript-bank/account.js

```diff
@@ -0,0 +1,44 @@
+/* exported Account */
+function Account(number, holder){
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+  return this;
```

Comment:
`this` is implicitly returned from a constructor function when the `new` keyword is used. In most cases we don't want to return anything from a constructor function it can cause undesired side effects. Please remove this line.